### PR TITLE
Archive optimizations cookbook

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
* archive optimisations cookbook
* Redirect cookbook URL to optimisations overview page
* move some useful content from ^ into the parallelism doc
* rename parallelism doc to Test Splitting and Parallelism to better encompass the feature
* split out test splitting troubleshooting into a separate page
* Tweak some text to get test splitting doc down to 5 minute read

To Do in separate tickets: 
* Split out "how to" content from Test Splitting and Parallelism doc so we have separate conceptual, how to and tutorial content.
* Write another tutorial on test splitting using environment variables without using the CLI

# Reasons
* Optimizations cookbook content was largely duplicated across other docs and has low count
* Parallelism doc needed a better title
* Parallism doc was too long

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: https://circleci.com/docs/style/.

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [x] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [x] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [x] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/parallelism-faster-jobs.md [hunk 1]
```diff
@@ -1,30 +1,86 @@
...
+## Test splitting to speed up pipelines
+{: #test-splitting-to-speed-up-pipelines }
+
+Pipelines are often configured so that each time code is committed a set of tests are run. Test splitting is a great way to speed up the testing portion of your CI/CD pipeline. A set of tests can be split over a range of test environments running in parallel.
+
+CircleCI test splitting lets you intelligently define where splits happen across a test suite: 
+
+* by name
+* by size
+* using timing data.
```
Comment (devinmarieb) [inline jekyll/_cci2/parallelism-faster-jobs.md:25]:
```suggestion
* using timing data
```

File: jekyll/_cci2/parallelism-faster-jobs.md [hunk 2]
```diff
@@ -95,18 +146,16 @@ jobs:
...
+On each successful run of a test suite, CircleCI saves timings data from the directory specified by the path in the [`store_test_results`]({{ site.baseurl }}/2.0/configuration-reference/#store_test_results) step. This timings data consists of how long each test took to complete per filename or classname.
 
-Note: If you do not use `store_test_results`, there will be no timing data available for splitting your tests.
+**Note**: If you do not use `store_test_results`, there will be no timing data available to split your tests.
 
-To split by test timings, use the `--split-by` flag with the `timings` split type. The available timings data will then be analyzed and your tests will be split across your parallel-running containers as evenly as possible leading to the fastest possible test run time
+To split by test timings, use the `--split-by` flag with the `timings` split type. The available timings data will then be analyzed and your tests will be split across your parallel-running containers as evenly as possible
```
Comment (devinmarieb) [inline jekyll/_cci2/parallelism-faster-jobs.md:158]:
```suggestion
To split by test timings, use the `--split-by` flag with the `timings` split type. The available timings data will then be analyzed and your tests will be split across your parallel-running containers as evenly as possible.
```

File: jekyll/_cci2/parallelism-faster-jobs.md [hunk 3]
```diff
@@ -193,107 +232,12 @@ bundle exec rspec $(cat /tmp/tests-to-run)
...
-
-### Video: troubleshooting globbing
-{: #video-troubleshooting-globbing }
-
-Note: To follow along with the commands in the video below you will need to be [`SSH-ed into a job`]({{ site.baseurl }}/2.0/ssh-access-jobs/).
+## Using environment variables to split tests
+{: #using-environment-variables-to-split-tests }
 
-<iframe width="854" height="480" src="https://www.youtube.com/embed/fq-on5AUinE" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+For full control over how tests are split across parallel executors, CircleCI provides two environment variables that you can use in lieu of the CLI to configure each container individually.
```
Comment (devinmarieb) [inline jekyll/_cci2/parallelism-faster-jobs.md:238]: Is "in lieu" a phrase we want to use? I don't know how friendly this is for international users.

**Review by devinmarieb — COMMENTED** [review]
Nice! I didn't review for technical accuracy because this area is mostly new to me, but I made a few suggestions/comments just for formatting.

File: jekyll/_cci2/parallelism-faster-jobs.md [hunk 4]
```diff
@@ -1,30 +1,86 @@
...
 
 * TOC
 {:toc}
 
+## Test splitting to speed up pipelines
+{: #test-splitting-to-speed-up-pipelines }
+
+Pipelines are often configured so that each time code is committed a set of tests are run. Test splitting is a great way to speed up the testing portion of your CI/CD pipeline. A set of tests can be split over a range of test environments running in parallel.
+
+CircleCI test splitting lets you intelligently define where splits happen across a test suite:
```
Comment (rosieyohannan) [inline jekyll/_cci2/parallelism-faster-jobs.md:21]:
```suggestion
CircleCI test splitting lets you intelligently define where splits happen across a test suite:
```

File: jekyll/_cci2/parallelism-faster-jobs.md (hunk 3 above)
Comment (rosieyohannan) [inline jekyll/_cci2/parallelism-faster-jobs.md:238]:
```suggestion
For full control over how tests are split across parallel executors, CircleCI provides two environment variables that you can use in place of the CLI to configure each container individually.
```

**Review by devinmarieb — APPROVED** [review]

**Pull request merged**